src/components/input-field/input-field.tsx

```diff
@@ -17,6 +17,7 @@ import {
     ARROW_UP_KEY_CODE,
     ENTER,
     ENTER_KEY_CODE,
+    ESCAPE,
```

Comment:
Don't you need to add this as well? `ESCAPE_KEY_CODE,`

Reply:
It's deprecated so I don't think so but we're doing it everywhere else so perhaps I should do it for that reason... :)

Reply:
😄 I have no clue what these are, but I just saw that it's used everywhere else as you say. Let's maybe keep it consistent, and in another issue remove all deprecated stuff?

Reply:
Added keycode as well ⚡